### DISCUSSION TITLE
Entity Not Updating

### DISCUSSION THREAD

**Discussion by pooptubesock** [cytech/Home-Assistant-wundergroundpws#263] [Q&A] [answered]
Sorry if this has been asked and answered, but I am not finding a resolution. My weather data does not update to HA. I have tried using an automation, though I'm not sure I'm doing it correctly. I've confirmed that my PWS is connected and updating regularly to WU, but the HA data only gets updated if I re-load the integration. 

Please help... totally lost noob.

**Comment by rudizl** [comment]
Same here, I'm using an automation, witch reloads pws every hour.

**Reply by pooptubesock** [reply]
I tried an automation to re-load every 5 minutes but that didn't work. I've now started down the rabbit hole of rtl_433 and MQTT so I don't have to rely on WU API.

**Comment by rudizl** [comment, marked as answer]
`alias: Reload PWS
description: ""
mode: single
triggers:
  - hours: /1
    trigger: time_pattern
conditions: []
actions:
  - metadata: {}
    data:
      entry_id: "{{ config_entry_id('weather.ivarna103') }}"
    action: homeassistant.reload_config_entry
`
This works for me.

**Reply by pooptubesock** [reply]
You're a life saver. I'm still going to work on rlt_433 and MQTT, but this is a good bandaid until I can get that up and running.

Thank you!!

**Reply by poli27** [reply]
> `alias: Reload PWS description: "" mode: single triggers:
> 
> * hours: /1
>   trigger: time_pattern
>   conditions: []
>   actions:
> * metadata: {}
>   data:
>   entry_id: "{{ config_entry_id('weather.ivarna103') }}"
>   action: homeassistant.reload_config_entry
>   `
>   This works for me.

Hello

It doesn't update for me either, where should I enter this? :) thx
